scripts/zowe-parse-yaml.sh

```diff
@@ -365,4 +395,7 @@ echo "  ZOWE_APIM_DISCOVERY_HTTP_PORT="$ZOWE_APIM_DISCOVERY_HTTP_PORT >> $LOG_FI
 echo "  ZOWE_APIM_GATEWAY_HTTPS_PORT="$ZOWE_APIM_GATEWAY_HTTPS_PORT >> $LOG_FILE
 echo "  ZOWE_SERVER_PROCLIB_MEMBER="$ZOWE_SERVER_PROCLIB_MEMBER >> $LOG_FILE
 echo "  ZOWE_SERVER_PROCLIB_DSNAME="$ZOWE_SERVER_PROCLIB_DSNAME >> $LOG_FILE
+echo "  KEYSTORE_DIRECTORY="KEYSTORE_DIRECTORY >> $LOG_FILE
+echo "  ZOWE_USER_ID="ZOWE_USER_ID >> $LOG_FILE
+echo "  ZOWE_GROUP_ID="ZOWE_GROUP_ID >> $LOG_FILE
```

Comment:
These need to be variable names with the '$' eg 
`echo "  KEYSTORE_DIRECTORY="$KEYSTORE_DIRECTORY >> $LOG_FILE`